x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/condition_entry_input/index.tsx

```diff
@@ -113,6 +114,18 @@ export const ConditionEntryInput = memo<ConditionEntryInputProps>(
       ];
     }, [os]);
 
+    const operatorOptions = useMemo<Array<EuiSuperSelectOption<string>>>(() => {
+      const dropDownOptions = [
+        OperatorEntryField.included,
+        OperatorEntryField.wildcard_caseless,
+      ].map((e) => ({
```

Comment:
That's perfect, just an opinion, I like to use always it is possible object destructuring:
```
.map((value) => ({
        dropdownDisplay: OPERATOR_TITLES[value],
        inputDisplay: OPERATOR_TITLES[value],
        value,
})
```

Reply:
Indeed, much neater 👍